contest/views.py

```diff
@@ -166,6 +179,9 @@ def delete(request, cid):
     if can_delete_contest(request.user, contest):
         deleted_cid = contest.id
         contest.delete()
+        message = 'Contest %s deleted!' % \
```

Comment:
In my opinion, warning message should appear before the contest deleted.
Moreover, it should double check whether the user really want to delete contest or not.